### PR TITLE
Dev: Sequence performance test page

### PR DESCRIPTION
**Pull request by psiinon**
A configurable set of sequence pages.

## Overview
A set of test sequence pages designed to make it easy to test the performance of the sequence add-on/

## Checklist
- [x] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [x] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

File: addOns/dev/src/main/java/org/zaproxy/addon/dev/seq/performance/SequencePage.java [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+            if (seqStepStr != null) {
+                try {
+                    seqStep = Integer.parseInt(seqStepStr);
+                } catch (NumberFormatException e) {
+                    // Ignore
+                }
+            }
+        }
+        if (seqUuid == null) {
+            // Its a new
```
Comment (kingthorin) [inline addOns/dev/src/main/java/org/zaproxy/addon/dev/seq/performance/SequencePage.java:96]: _____?

File: addOns/dev/src/main/zapHomeFiles/dev-add-on/seq/performance/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	<link href="/tutorial.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+<div class="roundContainer">
+	<h1>ZAP Test Server</H1>
+	<h2>Sequence Performence Test</h2>
+	
+	A sequence of forms that must be accessed in order.<br>
+	A unique id for each sequence is maintained in a hidden "seqId" parameter.<br>
+	The sequence step id is held both in a hidden "seqStep" parameter and is associated wit the sequence id on
```
Comment (kingthorin) [inline addOns/dev/src/main/zapHomeFiles/dev-add-on/seq/performance/index.html:14]: wit ➡️ with

**Comment by psiinon** [comment]
Updated - those issues should be fixed, but I've also added another config option 😁

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**